### DISCUSSION TITLE
Quick question about logging

### DISCUSSION THREAD

**Discussion by nicholasyoannou** [qdm12/gluetun#1074] [Q&A] [answered]
How do I turn off logging? If I leave the docker instance on for too long, I end up with 16GB massive .log files. Massive files and I have limited storage on my machines. Is there a way to turn this off?

The file is called `1afb3cc9b8446236e6073b589d46b56c80fb0831c82a.....-json.log` sitting in `/var/lib/docker/containers/1afb3cc9b8446236e6073b589d46b56c80fb083.....`. 

I've already got HTTP_PROXY on and have turned logging for that off with the HTTPPROXY_LOG=off argument.

**Comment by qdm12** [comment, marked as answer]
Wait what are the logs? That sounds strange you have so much logs. I have mine running for months with only like a megabyte or so.

Otherwise, you can turn off logging through docker/docker-compose by setting the log driver to none. See https://stackoverflow.com/a/34591999

**Reply by nicholasyoannou** [reply]
I don't use docker compose files at all, I've been using the straight-up command. e.g. `docker run -d -it --rm --cap-add=NET_ADMIN -e VPN_SERVICE.....`
How would I put logging in there?

**Reply by nicholasyoannou** [reply]
Found the answer! As simple as adding `--log-driver none`, thanks for telling me! Don't use docker too much cos of my inexperience, thanks for the help :)
_Source for log-driver documentation: https://docs.docker.com/config/containers/logging/configure/_

**Reply by qdm12** [reply]
Nice, although please have a look at the logs. It's really not normal you got GBs of logs.
